ref/general/oidc.adoc

```diff
@@ -0,0 +1,77 @@
+:page-layout: general-reference
+:page-type: general
+:page-description: OpenID Connect (OIDC) is an identity protocol based on OAuth 2.0 that is widely used in SSO technologies. OIDC works well with cloud, mobile, and native applications. 
+:page-categories: OpenID Connect
+:seo-title: OpenID Connect (OIDC) is an identity protocol based on OAuth 2.0.
+:seo-description: OpenID Connect (OIDC) is an identity protocol based on OAuth 2.0 that is widely used in SSO technologies. OIDC works well with cloud, mobile, and native applications.
+
+= Configuring Open Liberty as an SSO provider in a dev/test environment
+
+In an increasingly fragile world of the internet, user credentials are precious resources that need to be protected. With single sign-on (SSO), users can log in easily without worrying that that their credentials are being exposed. Open Liberty helps you with identity management.
+
+You can configure Open Liberty as an SSO provider, so that web users can authenticate once when accessing Liberty resources such as HTML, JavaServer Pages (JSP) files, and servlets, or accessing resources in multiple Liberty servers that share the same Lightweight Third Party Authentication (LTPA) keys.
+
+== OpenID Connect
+
+OpenID Connect (OIDC) is a simple identity protocol based on OAuth 2.0 that is widely used in SSO technologies. OIDC works well with cloud, mobile, and native applications. OIDC lets a client  request the identity of the user as an ID token in a standardized, REST-like manner.
+
+The client application operates as an OpenID Connect Relying Party (RP) that relies on Open Liberty as the OpenID Connect Provider (OP) for user authentication. The RP receives user identity information from the OP and passes the identity information in the ID token to the back-end Open Liberty application server in the HTTP Request header. The application server can extract user identity information from the HTTP header to perform further authorization.
+
+Using Open Liberty as an OP provides the following benefits,
+
+- Open Liberty OP can be integrated with the Liberty LDAP user registry.
+- Open Liberty OP can be configured as token services by delegating user authentication to SAML identity provider or another OpenID Connect provider
```

Comment:
missing article:
delegating user authentication to **a** SAML identity provider or another OpenID Connect provider